src/hooks/useMixpanelPortfolioTracking/useMixpanelPortfolioTracking.tsx

```diff
@@ -42,11 +51,26 @@ export const useMixpanelPortfolioTracking = () => {
 
     const mp = getMixPanel()
     if (!mp) return
-    // set this against the user
-    mp.people.set(anonymizedPortfolio) // TODO(0xdef1cafe): restructure multiple wallets per user
+    mp.people.set({
+      // set this against the user
+      anonymizedPortfolio,
```

Comment:
you'll need to `Object.assign` this otherwise `anonymizedPortfolio` will become nested and break all the mixpanel reports

Reply:
Right, thanks!